workspace/assets/js.json

```diff
@@ -54,7 +62,10 @@
 		"https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.3/jquery.easing.min.js",
 		"https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js",
 		"https://cdnjs.cloudflare.com/ajax/libs/underscore.string/3.3.4/underscore.string.min.js",
-		"https://cdnjs.cloudflare.com/ajax/libs/velocity/1.5.0/velocity.min.js"
+		"https://cdnjs.cloudflare.com/ajax/libs/velocity/1.5.0/velocity.min.js",
+		"https://www.google.com/recaptcha/api.js?onload=GoogleReCaptchaCallback&amp;render=explicit",
```

Comment:
They should not be there by default

Reply:
No please remove. It is also not at the right place.